packages/trpc/server/routers/loggedInViewer/me.handler.ts

```diff
@@ -24,7 +25,7 @@ export const meHandler = async ({ ctx }: MeOptions) => {
     locale: user.locale,
     timeFormat: user.timeFormat,
     timeZone: user.timeZone,
-    avatar: `${WEBAPP_URL}/${user.username}/avatar.png`,
+    avatar: `${userOrgDomain}/${user.username}/avatar.png`,
```

Comment:
There is `getBookerUrl` already that does this job. But that is async and does a query to fetch the org from orgId.

Maybe we can create a synchronous `getBookerUrl` which can accept the org directly and return the bookerUrl.

This is important because there is more logic in it than `getUserOrgDomain` has currently. It needs to check for `requestedSlug` as well.